internal/tools/terraform/plan.go

```diff
@@ -0,0 +1,143 @@
+package terraform
+
+import (
+	"encoding/json"
+	"fmt"
+
+	// "io/ioutil"
+	// "path"
+	"strings"
+
+	"github.com/mattermost/mattermost-cloud-database-factory/model"
+
+	"github.com/pkg/errors"
+)
+
+type terraformOutput struct {
+	Sensitive bool        `json:"sensitive"`
+	Type      string      `json:"type"`
+	Value     interface{} `json:"value"`
+}
+
+// Init invokes terraform init.
+func (c *Cmd) Init(remoteKey string) error {
+	// input, err := ioutil.ReadFile(path.Join("terraform", backendFilename))
```

Comment:
Not needed?

Reply:
Yeap, I removed the unused part thanks :)